packages/is-shallow-equal/package.json

```diff
@@ -18,8 +18,11 @@
 	"bugs": {
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
-	"main": "build/index.js",
-	"module": "build-module/index.js",
+	"main": "index.js",
+	"files": [
+		"arrays.js",
+		"objects.js"
+	],
```

Comment:
package.json linting failed with this:

```
./packages/is-shallow-equal/package.json
[0] ✖ prefer-property-order - node:  - Your package.json properties are not in the desired order. Please move "main" after "files".
```

Reply:
Ha, correct. Our linter is very strict 😄